packages/netlify-cms-widget-media/src/withFileControl.js

```diff
@@ -0,0 +1,286 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ImmutablePropTypes from 'react-immutable-proptypes';
+import styled from '@emotion/styled';
+import { Map, List } from 'immutable';
+import { once } from 'lodash';
+import uuid from 'uuid/v4';
+import { oneLine } from 'common-tags';
+import { lengths, components, buttons, borders, effects, shadows } from 'netlify-cms-ui-default';
+import { basename } from 'netlify-cms-lib-util';
+
+const MAX_DISPLAY_LENGTH = 50;
+
+const ImageWrapper = styled.div`
+  flex-basis: 155px;
+  width: 155px;
+  height: 100px;
+  margin-right: 20px;
+  margin-bottom: 20px;
+  border: ${borders.textField};
+  border-radius: ${lengths.borderRadius};
+  overflow: hidden;
+  ${effects.checkerboard};
+  ${shadows.inset};
+`;
+
+const StyledImage = styled.img`
+  width: 100%;
+  height: 100%;
+  object-fit: contain;
+`;
+
+const Image = props => <StyledImage role="presentation" {...props} />;
+
+const MultiImageWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const FileLink = styled.a`
+  margin-bottom: 20px;
+  font-weight: normal;
+  color: inherit;
+
+  &:hover,
+  &:active,
+  &:focus {
+    text-decoration: underline;
+  }
+`;
+
+const FileLinks = styled.div`
+  margin-bottom: 12px;
+`;
+
+const FileLinkList = styled.ul`
+  list-style-type: none;
+`;
+
+const FileWidgetButton = styled.button`
+  ${buttons.button};
+  ${components.badge};
+`;
+
+const FileWidgetButtonRemove = styled.button`
+  ${buttons.button};
+  ${components.badgeDanger};
+  margin-top: 12px;
+`;
+
+function isMultiple(value) {
+  return Array.isArray(value) || List.isList(value);
+}
+
+const warnDeprecatedOptions = once(field =>
```

Comment:
Not sure we need this deprecation notice in the new widget

Reply:
Not sure why this happened as I just checked and I added a deprecation notice to only the file and image widget. Will have to look more into this.